rust/src/ssh/ssh.rs

```diff
@@ -64,7 +65,8 @@ pub enum SSHEvent {
 }
 
 #[repr(u8)]
-#[derive(Copy, Clone, PartialOrd, PartialEq, Eq)]
+#[derive(AppLayerState, Copy, Clone, PartialOrd, PartialEq, Eq)]
+#[suricata(alstate_strip_prefix = "SshState")]
```

Comment:
Any thoughts on how and where we can document stuff like this? It shouldn't hold up this PR, but go on a documentation todo list.

Reply:
Document the derive `AppLayerState` usage ? In the dev guide @jufajardini ?